src/snowflake/cli/plugins/nativeapp/version/version_processor.py

```diff
@@ -155,13 +152,10 @@ def process(
         # Make sure version is not None before proceeding any further.
         # This will raise an exception if version information is not found. Patch can be None.
         if not version:
-            log.info(
-                dedent(
-                    f"""\
-                        Version was not provided through the CLI. Checking version in the manifest.yml instead.
-                    """
-                )
+            cc.step(
```

Comment:
another one that's not a step and not a warning.

Reply:
agreed, keeping step as a replacement to info.